templates/semantic-ui/invenio_app_rdm/header.html

```diff
@@ -6,130 +6,133 @@
 #}
 {%- block site_banner %}
 {% endblock site_banner %}
-{%  block heroheader %}
-<div>
-  <header class="theme header">
-    <div class="gradient-blue"></div>
-    <div class="gradient-teal"></div>
-    <div class="ui container glass-box">
-      <div class="glass-box"></div>
-    </div>
-    <div class="outer-navbar ">
+{% block heroheader %}
+  <div>
+    <header class="theme header">
+      <div class="outer-navbar">
 
-      {%- block navbar %}
-        <div class="ui container invenio-header-container">
-        <nav id="invenio-nav" class="ui menu inverted stackable p-0">
-          <div class="borderless item">
-            {%- block navbar_header %}
-              {%- block brand %}
-                {%- if config.THEME_LOGO %}
-                  <a class="logo-link" href="/">
-                    <img class="ui image rdm-logo"
-                         src="{{ url_for('static', filename=config.THEME_LOGO) }}"
-                         alt="{{ _(config.THEME_SITENAME) }}"/>
-                  </a>
-                {%- else %}
-                  <a class="logo" href="/">{{ _(config.THEME_SITENAME) }}</a>
-                {%- endif %}
-              {%- endblock brand %}
-            {%- endblock navbar_header %}
-          </div>
+        {%- block navbar %}
+          <div class="ui container fluid invenio-header-container">
+          <div class="ui container fluid menu-background">
+          <div class="ui container">
+          <nav id="invenio-nav" class="ui menu inverted stackable p-0">
+            <div class="borderless item">
+              {%- block navbar_header %}
+                {%- block brand %}
+                  {%- if config.THEME_LOGO %}
+                    <a class="logo-link" href="/">
+                      <img class="ui image rdm-logo"
+                           src="{{ url_for('static', filename=config.THEME_LOGO) }}"
+                           alt="{{ _(config.THEME_SITENAME) }}"/>
```

Comment:
```suggestion
                           alt="{{ _(config.THEME_SITENAME) }} home"/>
```
To indicate it's a home-link